source/boards/migrate-to-boards.rst

```diff
@@ -1,9 +1,31 @@
-Migrate to boards
-=================
+Import, export, and migrate
+===========================
 
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
+Import and export a board archive
+---------------------------------
+
+If you’d like to back up your board or re-use it on another team or channel workspace, you can export it as an archive file, and then import the archive file in the team or channel workspace of your choosing. Exported and imported board archives include all card content such as properties, comments, descriptions, and image attachments.
+
+To do this, select the options menu **(…)** to the left of the **New** button at the top of the board. Then select **Export board archive**. Download the archive file. Navigate to the team or channel workspace where you’d like to add the exported board. Select the Gear icon next to your profile avatar, then choose Import archive. The board you exported will be added to this team or channel workspace.
```

Comment:
Incorporating the options menu icon would work the same way as the Gear icon. See the [Mark messages as unread](https://docs.mattermost.com/channels/mark-messages-unread.html) docs page and source file as a working example.

This is another great page to incorporate the Gear icon into too.

Given that an export and import workflow includes multiple sub-steps, would it make sense to split this into a numbered list rather than a paragraph?